aio/aio-proxy/aio_proxy/search/person.py

```diff
@@ -0,0 +1,116 @@
+from elasticsearch_dsl import query
+
+
+def search_person(
+    search,
+    param_nom,
+    param_prenom,
+    param_date_min,
+    param_date_max,
+    properties,
+    **params,
+):
+    search_options = []
+    for p in properties:
```

Comment:
`properties` is very vague

Reply:
Since this function is a little different than the "dirigeants search" one, I think a more elaborate explanation is in order.